protocol-definitions/custom/Custom.yaml

```diff
@@ -1056,3 +1056,63 @@ customTypes:
         type: DiskTierConfig
         nullable: false
         since: 2.5
+  - name: JobStatus
```

Comment:
Is there a reason we have this as a custom type definition? I believe we can make it an `int` field of `JobAndSqlSummary`

Reply:
Yes, it will stay as an enum in the Java code. We will make it an `int` parameter here, and add a method to https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/builtin/FixedSizeTypesCodec.java for it to do the conversion

Reply:
See this for example https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/builtin/FixedSizeTypesCodec.java#L120

Reply:
Done